BUILD.md

```diff
@@ -114,12 +114,6 @@ Device backends can be enabled by specifiying `-DKokkos_ENABLE_X`.
 * Kokkos_ENABLE_PTHREAD
     * Whether to build Pthread backend
     * BOOL Default: OFF
-* Kokkos_ENABLE_QTHREAD
-    * Whether to enable the QTHREAD library
-    * BOOL Default: OFF
```

Comment:
Do we actually want to remove the `QTHREAD` and `ROCM` backend completely for the release or just deprecate them?